TableController/TableController/Sources/SectionController.swift

```diff
@@ -106,18 +106,46 @@ open class SectionController: NSObject {
         }
     }
 
+    open func insertCellControllers(_ cellControllers: [CellController], at indices: [Int], with animation: UITableViewRowAnimation = .automatic) {
```

Comment:
I considered that, but I find `insert(stuff: [Thing], at indices:[Int])` aka "insert stuff at places" easier to read/interpret than `insert(stuff:[(aThing: Thing, index: Int)])` aka "insert stuff, but pack it up in this tuple first"

It also loses symmetry with the deletion function.

If it were going to change, I would almost rather make it a singular operation rather than a batch operation. Though that blocks the consumer from using that UITableView capability, which is gross.

Reply:
I do find `insert(stuff: [Thing], at indices:[Int])` easier to read, actually. I just worry about providing a user-proof interface. 

We should at very least make some assertions about the arrays matching length, and the indices being in bounds.

Reply:
Why not let UITableView do that for us, since it's going to do that anyways?

Reply:
Fair, but the `zip` will swallow the difference of the larger array if they don't match. We should check that part at least.

Reply:
Added an assertion in 3b76ef9 that ensures the arrays have equal counts